kensu/utils/kensu.py

```diff
@@ -71,7 +73,8 @@ def __init__(self, api_url=None, auth_token=None, process_name=None,
         from configparser import ConfigParser, ExtendedInterpolation
 
         config = ConfigParser(interpolation=ExtendedInterpolation())
-        # TODO... path to conf there are so many args in the function here, so adding it will require a good migration plan (it doesn't land in kwargs...)
+        # TODO... path to conf there are so many args in the function here, so adding it will require a good migration plan 
+        # (it doesn't land in kwargs...)
```

Comment:
what does it mean `(it doesn't land in kwargs...)` ?

Reply:
I think we can remove the comment... in fact, the path to the conf file is handled as a var env, so it is not hardcoded anymore.

What I want to avoid is yet another kwargs variable that specifies the path to the conf file

The drawback is that the client to restart with a new conf requires a dedicated file set at that location. Which may prevent "re_init" the client easily.

Maybe it would be better to have the config loaded elsewhere and provide the loaded config to the Kensu (re)init instead... but adding a new parameter to `__init__` requires some thoughts due to the amount of args and kwargs already (that should be deprecated in favor of the conf in fact...)

Reply:
I will propose something else with a variable before kwargs... it should work I guess as long as a default value is provided